docs/use-cases/testing-in-production.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Testing in production (UAT)
+description: Learn how to coordinate user acceptance testing (UAT) in production with Featurevisor
+ogImage: /img/og/docs-use-cases-testing-in-production.png
+---
+
+As your application grows more complex with critical features, you want to have more confidence that everything works as expected in production environment where your real users are, before any new features are  exposed them. Coordinating user acceptance testing (UAT) with Featurevisor can help you there. {% .lead %}
+
+## What is User Acceptance Testing?
+
+User Acceptance Testing (UAT) is a type of testing performed by the end users or the client to verify/accept the software system before moving the software application to the production environment.
+
+UAT is done in the final phase of testing after functional, integration and system testing is done.
+
+## Who performs the testing?
```

Comment:
I would be very cautious on mixing QAs with SDETs. 
QAs are considered manual testers, that also deal with documenting test cases, and even tracking if the dev team has dealt with all the acceptance criteria without compromising existing functionality (regression).

SDETs on the other hand are heavily focused on automation. These are the folks that will try to introduce automated testing into every step of you CI, including UAT.
Frankly I haven't checked how the framework works and whom is audience

Reply:
@anskourt: I have updated the document further by distancing from UAT, and keeping the flow to QAs only without mentioning SDETs anywhere.